tests/sndrcv_relp_tls_certvalid.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# added 2018-08-28 by PascalWithopf
+# This file is part of the rsyslog project, released under ASL 2.0
+echo ===============================================================================
+echo \[sndrcv_relp_tls.sh\]: testing sending and receiving via relp with TLS enabled
```

Comment:
name in comment is wrong. I suggest to remove line 4+5, the testbench plumbing emits the name itself.